templates/index.html

```diff
@@ -148,61 +148,67 @@ <h4>3rd Party Software</h4>
         <div class="right">
           <div class="input__less">
             <label class="checkbox">
-              <input type="checkbox" value="chromium" name="INSTALL_chromium">
+              <input type="checkbox" value="chromium" name="INSTALL">
               <span>Chromium</span>
             </label>
           </div>
           <div class="input__less">
             <label class="checkbox">
-              <input type="checkbox" value="firefox" name="INSTALL_firefox">
+              <input type="checkbox" value="firefox" name="INSTALL">
```

Comment:
@rpotter12 this is a redunant change will not have any affect on the working of the app, I think different names provide more clarity to the packages being installed